### PR TITLE
[VE] Intrinsics for vector logical 2-operand instructions

### PR DESCRIPTION
**Pull request by akmaru**
Added the builtins and  LLVM-IR intrinsics for vpcnt, vldz and vbrv instructions.

### CODE REVIEW

**Comment by LWisteria** [comment]
We've used vpcnt on our project. This PR works fine!

File: llvm/lib/Target/VE/veintrin.py [hunk 1]
```diff
@@ -970,19 +970,34 @@ def Inst3divbys(self, opc, name, instName, subop, ty):
...
+
+        O_u64 = self.addMask(O_u64)
+        O_u32 = self.addMask(O_u32)
+        O_pu32 = self.addMask(O_pu32, VM512)
+
+        self.Def(opc, instName, "", name, O_u64).noTest()
+        self.Def(opc, instName, "p", "p" + name + ".lo", O_u32).noTest()
+        self.Def(opc, instName, "p", "p" + name + ".up", O_u32).noTest()
+        self.Def(opc, instName, "p", "p" + name, O_pu32).noTest()
+
```
Comment (kaz7) [inline llvm/lib/Target/VE/veintrin.py:986]: Please remove spaces here.

File: llvm/lib/Target/VE/veintrin.py [hunk 2]
```diff
@@ -970,19 +970,34 @@ def Inst3divbys(self, opc, name, instName, subop, ty):
...
 
-        self.Def(opc, instName, "", name, Args, expr)
-        #self.Def(opc, instName, "p", "p"+name+".lo", ArgsP, expr).noTest()
-        #self.Def(opc, instName, "p", "p"+name+".up", ArgsP, expr).noTest()
-        self.Def(opc, instName, "p", "p"+name, ArgsP, expr)
+        self.Def(opc, instName, "", name, O_u64, expr)
+        # self.Def(opc, instName, "p", "p"+name+".lo", O_pu32, expr).noTest()
+        # self.Def(opc, instName, "p", "p"+name+".up", O_pu32, expr).noTest()
+        self.Def(opc, instName, "p", "p" + name, O_pu32, expr)
+
```
Comment (kaz7) [inline llvm/lib/Target/VE/veintrin.py:1000]: Please remove this line (modify two empty lines into single empty line and remove spaces from empty lines)

**Review by kaz7 — CHANGES_REQUESTED** [review]
Thank you for contribution.  I've checked modified llvm for ve using internal test suites, and it works fine.  Thank you so much.  I need to ask you to fix a few typo.  May I ask you to remove these spaces?

**Comment by kaz7** [comment]
Well, I've merged and cleaned code like above description.  Thank you very much for your contributions.

**Comment by akmaru** [comment]
Thank you reviewing and fix!

**Pull request merged**